packages/api/db/migration/20220207164158_notification.js

```diff
@@ -0,0 +1,35 @@
+exports.up = async function (knex) {
+  await knex.schema.createTable('notification', function (table) {
+    table.uuid('notification_id').primary();
+    table.string('title').notNullable();
+    table.string('body').notNullable();
```

Comment:
Why not? The requirements are a bit fuzzy, but the idea was that it might hold text, just a translation tag, or text with embedded tags. I'm open to suggestions for something better.